server/login-by-phone.js

```diff
@@ -1,15 +1,16 @@
 Meteor.methods({
   loginByPhone: function(code, phone){
-    user = Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'profile.phone': phone});
+    user = Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'profile.phone': phone}) ||
+      Meteor.users.findOne({'services.phone.verificationTokens.code': code, 'phone': phone});
 
```

Comment:
that's a good idea! that will make it flexible for other schemas. Do you want me to change it?